
This is dirty hack needed to stop kernel oops on rk3528.
Root casue is under investigation.... 

diff --speed-large-files --no-dereference --minimal -Naur linux-6.4.11/drivers/soc/rockchip/pm_domains.c linux-6.4.11/drivers/soc/rockchip/pm_domains.c
--- linux-6.4.11/drivers/soc/rockchip/pm_domains.c	2023-08-18 15:16:20.963385947 +0200
+++ linux-6.4.11/drivers/soc/rockchip/pm_domains.c	2023-08-18 15:10:06.586718036 +0200
@@ -499,7 +499,7 @@
 			rockchip_pmu_save_qos(pd);
 
 			/* if powering down, idle request to NIU first */
-			rockchip_pmu_set_idle_request(pd, true);
+//			rockchip_pmu_set_idle_request(pd, true);
 		}
 
 		rockchip_do_pmu_set_power_domain(pd, power_on);
